smart_control/notebooks/SAC_Demo.ipynb

```diff
@@ -1658,44 +1534,12 @@
     },
     {
       "cell_type": "code",
-      "execution_count": 12,
+      "execution_count": null,
       "metadata": {
         "cellView": "form",
         "id": "3Zv-lSiWDp50"
       },
-      "outputs": [
```

Comment:
I wonder if we should be saving the notebook without outputs (to help with managing the code, in an effort to reduce the diffs).